BingMaps/v8-web-control/map-control-api/animatedframeeventargs-object.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "AnimatedFrameEventArgs Object | Microsoft Docs"
+description: "This article provides the event arguments for when a layer frame is being loaded in an AnimatedTileLayer."
```

Comment:
Let's add the API to the description, so that if someone is searching by API, they'll be able to find this article.  Maybe something like this:

"The AnimatedFrameEventArgs Object provides the event arguments for when a layer frame is being loaded in an AnimatedTileLayer."